spec/services/planning_application_search_spec.rb

```diff
@@ -0,0 +1,300 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+RSpec.describe PlanningApplicationSearch do
+  let(:search) { described_class.new(params) }
+
+  let!(:local_authority) { create(:local_authority, :default) }
+
+  let!(:assessor) do
+    create(:user, :assessor, local_authority:)
+  end
+
+  before do
+    allow_any_instance_of(described_class).to receive(:current_user).and_return(assessor)
```

Comment:
```suggestion
    allow(Current).to receive(:user).and_return(assessor)
```

I think it's probably bad practice to stub the class that's being tested. It looks like the `current_user` method just returns `Current.user`, so I think we should be able to stub that instead?

Reply:
In some specs we just do `before { Current.user = assessor }`